server/src/mds3.gdc.js

```diff
@@ -68,18 +67,15 @@ export async function validate_ssm2canonicalisoform(api, getHostHeaders) {
 		// q is client request object
 		if (!q.ssm_id) throw '.ssm_id missing'
 		const { host, headers } = getHostHeaders(q)
-		const response = await got(host.rest + '/ssms/' + q.ssm_id + '?fields=' + fields.join(','), {
-			method: 'GET',
-			headers
-		})
-		let re
-		try {
-			re = JSON.parse(response.body)
-		} catch (e) {
-			throw 'invalid json in response'
-		}
-		if (!re.data || !re.data.consequence) throw 'returned data not .data.consequence'
-		if (!Array.isArray(re.data.consequence)) throw '.data.consequence not array'
+		const re = await ky(
+			path.join(
+				host.rest,
+				'ssms',
+				q.ssm_id + '?fields=consequence.transcript.is_canonical,consequence.transcript.transcript_id'
```

Comment:
@xzhou82 If we have >1 URL params (not in this case), I think we should use a `searchParams` object here, instead of manually constructing URL params.